2-advanced-topics/data-pipelines/README.md

```diff
@@ -0,0 +1,60 @@
+        * APIs
+            * [Slides](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/Introduction%20to%20APIs.pdf)
+            * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb)
+            * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb)
+            * [Suggested solutions](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/foundations-s4-api-solutions.ipynb)
+        * Data visualization
+            * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb)
+            * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb)   
+    * Project 1 - Schelling model
+        * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/project-1-schelling-model/project-1-schelling-model-of-segregation.ipynb)
+        * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/1-foundations/project-1-schelling-model/project-1-schelling-model-of-segregation.ipynb)
+        * [Suggested solutions](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/project-1-schelling-model/project1-solutions.ipynb)
+    * Project 2 - Raster data analysis and ML inference
+        * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/project-2-rasters_and_functions/Project%202.ipynb)
+        * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/1-foundations/project-2-rasters_and_functions/Project%202.ipynb)
+        * [Suggested solutions](https://github.com/worldbank/dec-python-course/blob/main/1-foundations/project-2-rasters_and_functions/Project%202_Solutions.ipynb)
+* **Part 2 - Advanced topics of Data Science of Python**
+    * Geospatial analysis
+        * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/2-advanced-topics/geospatial-analysis/geospatial-analysis.ipynb)
+        * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/2-advanced-topics/geospatial-analysis/geospatial-analysis.ipynb)
+    * Introduction to text analysis
+        * [Jupyter notebook](https://github.com/worldbank/dec-python-course/blob/main/2-advanced-topics/text-analysis/intro-text-analysis.ipynb)
+        * [Colab notebook](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/2-advanced-topics/text-analysis/intro-text-analysis.ipynb)
+
+## Contact
+
+If you have any questions you can contact one of the teams behind this training
+on dimeanalytics@worldbank.org.
```

Comment:
@bhupatiraju this README seems identical to what's currently at the root of this repo. Did you mean to copy and modify?